docs/docs/development/api.md

```diff
@@ -550,7 +550,7 @@ There is a XML response for this call only when the `redirect` parameter is set
 </response>
 ```
 
-See [Passing custom parameters to the client on join](/administration/customize/#passing-custom-parameters-to-the-client-on-join) for additional `join` parameters that can override default settings for the user.
+See [Passing user metadata to the client on join](/administration/customize/#passing-custom-parameters-to-the-client-on-join) for additional `join` parameters that can override default settings for the user.
```

Comment:
```suggestion
See [Passing user metadata to the client on join](/administration/customize/#passing-user-metadata-to-the-client-on-join) for additional `join` parameters that can override default settings for the user.
```

Please search and update the docs for other links to the same.

Reply:
Looks like this was the only one. Thanks!